src/merge_outputs.rs

```diff
@@ -25,6 +26,7 @@ pub struct LayerUploads {
 pub async fn merge(
     merge_config: &super::merge_config::MergeConfig,
     relative_search_path: &Option<PathBuf>,
+    external_execution_config: &Option<ExecutionConfig>,
```

Comment:
seems like this could be a `&Vector<ExecutionConfig>` and just iterate through updating?

Reply:
Yeah, re: the comment on your other PR, if we're going to be able to have multiple file configs, this will need to be a vector. 

there is an ExecutionConfig buried in the MergeConfig passed in here, and that one needs to be the last merge, so its fine to leave it as-is.